netshare/utils/word2vec_utils.py

```diff
@@ -0,0 +1,9 @@
+import os
```

Comment:
I think that this is just an empty file?

Reply:
Yes, sorry about that:)